### ISSUE TITLE
Manual and automated testing for monitoring mechanism

### ISSUE THREAD

**Issue by otsakir** [RestComm/visual-designer#204]
Write automated arquilian tests for ES monitoring mechanism. Also, do manual testing for corner cases.

Relates to #184

**Comment by otsakir** [comment]
Issue is blocked by #207 . At least for automated testing.

**Comment by otsakir** [comment]
Performed manual tests succesfully.

**Comment by otsakir** [comment]
A separate issue should be created for automated testing once refactoring has finished - #212
